src/main/java/com/google/sps/data/SiteUser.java

```diff
@@ -0,0 +1,68 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import com.google.appengine.api.datastore.Key;
+
+public class SiteUser {
+    private String displayName;
+    private String email;
+    private Key key;
+    private String imageID;
```

Comment:
Does it make sense for these Ids to be GUIDs? Are they values generated/returned from the database?